plugins/accuracy.cc

```diff
@@ -251,8 +252,12 @@ runCategorical(AccuracyConfig & runAccuracyConf,
                std::shared_ptr<Dataset> output,
                int accuracyOverN)
 {
-    typedef vector<std::tuple<CellValue, CellValue, std::vector<CellValue>, double>> AccumBucket;
+    typedef vector<std::tuple<CellValue, CellValue, double>> AccumBucket;
+    typedef vector<std::tuple<CellValue, CellValue, std::vector<CellValue>, double>> AccumBucketWithBestLabels;
```

Comment:
The inner vector should be compact_vector<CellValue, 1> as normally there will only be one best label, and this avoids memory allocations in that case.